projects/packages/my-jetpack/_inc/data/evaluation-recommendations/use-evaluation-recommendations.ts

```diff
@@ -111,13 +111,18 @@ const useEvaluationRecommendations = () => {
 		);
 	}, [ handleRemoveEvaluationResult, setRecommendedModules, showWelcomeBanner ] );
 
+	const unownedRecommendedModules = useMemo( () => {
+		const { unownedProducts = [] } = getMyJetpackWindowInitialState( 'lifecycleStats' );
+		return recommendedModules?.filter( module => unownedProducts.includes( module ) );
+	}, [ recommendedModules ] );
```

Comment:
One more thing about the filtering out owned products. I know that the recommendations endpoint is set to return the top 3 recommendations 🤔 But if the user owns a recommendation or two, this leaves a less-than-full recommendations section. This is more of a question rather than a suggestion, but I wonder if it would be better to set the recommendations endpoint to return all the products ranked highest to lowest so we can still populate the top 3 if the user has any or all of the regular top 3 active already?

I am not sure if that's the right way to go because we could end up with recommendations that don't make sense for what they picked. My opinionated ideal solution would be to return all products ranked in the endpoint and populate the list as much as possible, but have a minimum score needed for a product to show up in the recommendations list 🤔 This all may be something we should think of as an i2 iteration though and not to be worried about in i1

Reply:
> This is more of a question rather than a suggestion, but I wonder if it would be better to set the recommendations endpoint to return all the products ranked highest to lowest so we can still populate the top 3 if the user has any or all of the regular top 3 active already?

Yes! That's exactly what I'm having planned as a second round of clean up tasks :D 

> but have a minimum score needed for a product to show up in the recommendations list
This is a very good point, thanks!